src/ToolsUnix.cpp

```diff
@@ -0,0 +1,44 @@
+#ifdef __unix__
```

Comment:
The defines are still not fixed and you still include unix and windows tools files into every compilation.

Reply:
Awaiting @Cryptyc's decision on this one - will hold off changing it until then.

Reply:
I think its fine to leave it in here and just #define it out for other platforms.  Just put a comment in there to make it double obvious to anyone who reads it.

Reply:
>Just put a comment in there to make it double obvious to anyone who reads it.

Sorry @Cryptyc 
Do you mean something explaining the preprocessor directive, such as:
`// Only include for Unix systems`